microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -128,9 +129,16 @@ func createService(step string, ns string, svc string, svcCfg *map[string]string
 
 		gvr, _ := meta.UnsafeGuessKindToResource(*gvk)
 		for {
-			// _, err = dynamicClient.Resource(gvr).Namespace(ns).List(context.TODO(), metav1.ListOptions{})
+			patchBytes, err := json.Marshal(obj)
+			if err != nil {
+				return fmt.Errorf("failed to marshal config to json: %v", err)
+			}
 
-			createdObj, err := dynamicClient.Resource(gvr).Namespace(ns).Create(context.TODO(), obj, metav1.CreateOptions{})
+			// createdObj, err := dynamicClient.Resource(gvr).Namespace(ns).Create(context.TODO(), obj, metav1.CreateOptions{})
+			createdObj, err := dynamicClient.Resource(gvr).Namespace(ns).Patch(context.TODO(), obj.GetName(), types.ApplyPatchType, patchBytes, metav1.PatchOptions{
+				FieldManager: "your-field-manager",
```

Comment:
please change 'your-filed-manager' with our gmc controller name

Reply:
thanks for your comment, changed to "gmc-controller"